static/js/components/UserInfoCard.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+import React from 'react';
+import Grid, { Cell } from 'react-mdl/lib/Grid';
+import { Card } from 'react-mdl/lib/Card';
+import IconButton from 'react-mdl/lib/IconButton';
+
+import ProfileImage from './ProfileImage';
+import {
+  getEmployer,
+  getPreferredName,
+  userPrivilegeCheck
+} from '../util/util';
+import type { Profile } from '../flow/profileTypes';
+
+export default class UserInfoCard extends React.Component {
+  props: {
+    profile: Profile,
+    toggleShowPersonalDialog: () => void
+  };
+
+  getCurrentWorkPlace: Function = (profile: Profile): string => {
+    let companyName = "";
+    if (!getEmployer(profile).isNothing) {
+      companyName = getEmployer(profile).value;
+    }
+    return companyName;
+  }
```

Comment:
But this function isn't necessary anyway, `getEmployer` returns a `Maybe<string>`. There's a utility function in `util/sanctuary` called `mstr` that replicates this:

```js
const mstr = S.maybe("", String);
```

Basically, if you have a `Maybe` you can pass it to `mstr` and if the maybe is `Nothing` you'll get `""`, and otherwise you'll get `String(val)`, where `val` is the value wrapped up in the `Maybe`.

Here's an example:

```js
import { Just, Nothing } from 'sanctuary';
import { mstr } from '../util/sanctuary';

const justName = Just("Alice");

const nothing = Nothing();

mstr(justName) // -> "Alice"
mstr(nothing)  // -> ""
```

So you can call `mstr(getEmployer(profile))` inline in the JSX, and that should take care of everything!